operation/operation.c

```diff
@@ -110,7 +110,7 @@ void logf_msg(enum logs logEnum, oper_st * state, int n_args, ...) {
             write_to_eeprom(COMP_INDEX_ADDR,
                             &state->current_comp_idx, 1);
             if (state->lora_connected)
-                send_msg_to_Lora(msg, false);
+                send_msg_to_Lora(msg, true);
```

Comment:
One thing I'm not sure about is whether we need to get response from Lora after sending message? Is it in our requirement somehow or this is just our own decision that we want to sometimes check if we get response from lora when sending command?